auth-api/src/auth_api/utils/roles.py

```diff
@@ -25,6 +25,7 @@ class Role(Enum):
     EDITOR = 'edit'
     ADMIN = 'admin'
     SYSTEM = 'system'
+    POSTMAN = 'postman'
```

Comment:
May want to call this something more generic, like 'tester'

Reply:
I changed the path to '/test' and the role name to 'tester'.